### PR TITLE
feat: add grant moderator functionality

### PR DESCRIPTION
**Pull request by gimre**
related PR in lib-jitsi-meet:
https://github.com/jitsi/lib-jitsi-meet/pull/1239/files

### CODE REVIEW

File: lang/main.json [hunk 1]
```diff
@@ -668,6 +668,7 @@
             "e2ee": "End-to-End Encryption",
             "feedback": "Leave feedback",
             "fullScreen": "Toggle full screen",
+            "grantModerator": "Grant Moderator",
```
Comment (damencho) [inline lang/main.json:671]: Not something important and maybe out of the scope of this PR...
But I'm not sure why we need two identical strings for translation, isn't just one enough?

**Comment by damencho** [comment]
Make sure you plan adding torture tests for this new feature. Thanks.

**Comment by damencho** [comment]
Will tests be added for this feature?

**Comment by gimre** [comment]
Yes, hello! I've had some time off.

File: lang/main.json (hunk 1 above)
Comment (gimre) [reply lang/main.json:673]: It looks like it's future proofing, I see a lot of other components have this pattern where every text gets its own key, even though they might be the same in value now

**Pull request merged**